docs/concepts/trading-framework/trading-modes.md

```diff
@@ -0,0 +1,113 @@
+---
+sidebar_position: 1
+title: Trading modes
+hide_title: false
+---
+# Trading modes 
+A market's trading mode denotes the types of trading that can be done on it while the market is in that mode. A market can only have one trading mode at a time.  The Vega network currently supports futures markets.
+
+The Vega software currently supports two trading modes: continuous trading (using a limit order book) and auctions.
+
+## Continuous trading
+On a market with continuous trading, the Vega network tries to execute an order as soon as it is received. 
+
+A continuous trading market uses a limit order book as the default price determination method.
+
+Most, but not all, order types and times in force are accepted during continuous trading. 
+
+### Orders accepted during continuous trading
+
+| Pricing method | GTT | GTC | IOC | FOK | GFA | GFN |
+| -------------- |:---:|:---:|:---:|:---:|:---:|:---:|
+| Limit          | Y   | Y   | Y*  | Y*  | N   | Y   |
+| Pegged         | Y   | Y   | N** | N** | N   | Y   |
+| Market         | N   | N   | Y   | Y   | N   | N   |
+
+\* IOC/FOK LIMIT orders never rest on the book, if they do not match immediately they are cancelled/stopped.<br/>
+\** IOC/FOK PEGGED orders are not currently supported as they will always result in the cancelled/stopped state. This may change in the future if pegged orders are allowed to have negative offsets that can result in an immediate match.
+
+## Auctions
+Auctions are a trading mode that collect orders during a set period, called an [auction call period](#auction-call-period). The end of an auction call period is determined by the condition that the auction aims to meet.
+
+Auctions aggregate participation over time, up to a pre-set time when the market is uncrossed. The auction uncrossing generates trades at the conclusion of the auction, using an algorithm that processes, in price-time priority, the set of crossed orders that maximises traded volume. 
+
+Currently, all auctions are triggered automatically based on market conditions. Market conditions that could trigger an auction:
+* A market has opened for trading, which means it needs a defined price to start trading from 
+* Price swing on a market is perceived, based on risk models, to be extreme and unrealistic
+* Not enough liquidity on a market
```

Comment:
```suggestion
* Not enough liquidity on a market; this could also indicate missing best static bid / ask as without those liquidity provision orders cannot be deployed, even if liquidity providers have committed liquidity. 
```